app/Console/Commands/GeneratePdfs.php

```diff
@@ -71,7 +71,7 @@ protected function generatePdf($model, $route = null)
         if (empty($route)) {
             return false;
         }
-        $baseUrl = config('aic.protocol') . '://' . config('app.url');
+        $baseUrl = config('app.url');
```

Comment:
Laravel provides this functionality: https://laravel.com/docs/11.x/helpers#method-url

Reply:
We're not generating a full URL here, just passing the domain to be used for URLs in Prince, so this is abbreviated enough.